### PR TITLE
[tests-only] [full-ci] Added Then steps to improve checks in tests for apiWebdavProperties suites

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
This PR adds `Then` steps to existing API test scenarios to better check the results of `When` steps.

**Suites covered:**
- `apiWebdavProperties1`
- `apiWebdavProperties2`

## Related Issue

- Part of https://github.com/owncloud/core/issues/39512

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiWebdavProperties1/copyFile.feature [hunk 1]
```diff
@@ -77,8 +77,8 @@ Feature: copy file
     And user "Alice" has accepted share "/testshare" offered by user "Brian"
     When user "Alice" copies file "/textfile0.txt" to "/Shares/testshare/textfile0.txt" using the WebDAV API
     Then the HTTP status code should be "403"
-    And user "Alice" downloads file "/Shares/testshare/textfile0.txt" using the WebDAV API
-    And the HTTP status code should be "404"
+    When user "Alice" downloads file "/Shares/testshare/textfile0.txt" using the WebDAV API
+    Then the HTTP status code should be "404"
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiWebdavProperties1/copyFile.feature:81]: downloading seems not a when `action` here.
what about:
```feature
And user "Alice" should not be able to download "/Shares/testshare/textfile0.txt"
```

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

**Review by phil-davis — APPROVED** [review]

**Pull request merged**